src/compiler/diagnosticMessages.json

```diff
@@ -3451,6 +3451,10 @@
         "category": "Error",
         "code": 6189
     },
+    "Specify library files to be included in the compilation.": {
+        "category": "Message",

```

Comment:
Why add a new message? Just remove the colon from the other one.

Reply:
Sorry, I've just got confused when I was about to remove the colon because it also changes command-line message which list of allowed parameters is shown.